pom.xml

```diff
@@ -40,10 +40,8 @@
     <version.com.h2database>1.4.199</version.com.h2database>
     <version.javax.validation>2.0.1.Final</version.javax.validation>
     <version.org.assertj>3.11.1</version.org.assertj>
-    <version.org.mockito>2.23.4</version.org.mockito>
+    <version.org.mockito>3.1.0</version.org.mockito>
```

Comment:
@cuijulian Why this?

Reply:
The older version of Mockito resulted in test errors, with the same exception as in [this GitHub issue](https://github.com/mockito/mockito/issues/1604). Upgrading to the Mockito version that Spring Boot Test Starter uses (3.1.0) resolves this issue. As stated by one of the Spring Boot developers the problem was probably caused by a mixture of Mockito 2.23.4 with the Spring Boot version.